src/main.js

```diff
@@ -7,8 +7,9 @@ const express = require('express');
 const app = express();
 
 app.get('/', async function(request, response) {
-  const head = await render(request.query.url).catch((err) => console.error(err));
-  response.send(head);
+  const renderWebComponents = !!request.query['wc'];
```

Comment:
`forceShadydomPolyfill` or similar might be a better name than `renderWebComponents` or `usingWebComponents` since it's specifically the polyfill behavior that it's switching.